docs/msbuild/customize-your-build.md

```diff
@@ -86,6 +98,51 @@ A summary of MSBuild's general approach is as follows:
 
 Or more simply: the first *Directory.Build.props* which doesn't import anything, is where MSBuild stops.
```

Comment:
"First" here being the "deepest" file, the one firthest down in the directory structure?

Reply:
Correct. The "closest to the searching project"/"first to be found by following this search scheme".